2682. Find the Losers of the Circular Game/solution.py

```diff
@@ -0,0 +1,29 @@
+class Solution:
+    # taken the number of players, n, and a constant, k, it returns the losers of a circular game. player 1 starts with the ball,
+    # and passes it to the person k times away from him. The same thing for 2*k. Then 3*k. And so on until the player who has 
```

Comment:
k times what? Also you can't assume the person is male.

Reply:
Oh, sorry, I don't know what I was doing when I was writing it. It meant "players".